package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "node_ecourse",
+  "version": "0.0.1",
+  "description": "Node.js course tasks",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel ./app/server/ --out-dir dist",
+    "start": "babel ./app/server/ --watch --out-dir dist --source-maps inline",
+    "serve": "npm run build && node dist/index.js",
```

Comment:
For meet the requirements on task, try to move the script for development to `start:dev` and rename `serve` to `start` as script for production. This change will keep your own script with file watcher and the build will happen with a `npm start`.

Reply:
Done